packages/renderer/src/AppNavigation.spec.ts

```diff
@@ -31,6 +31,18 @@ beforeAll(() => {
   };
 });
 
+test('Test navigation bar gets right role and aria-label', () => {
+  render(AppNavigation, {
+    meta: {
+      url: '/',
+    },
+  });
+
+  const navigationBar = screen.getByRole('navigation', { name: 'AppNavigation' });
+  expect(navigationBar).toBeInTheDocument();
+  expect('AppNavigation').toEqual(navigationBar.getAttribute('aria-label'));
```

Comment:
not sure what it really checks ?

Reply:
I think the tests are wrongly named the aria-label is a technical concept used by the tests to locate items but the purpose of the tests should not to test if aria-labels are working

Reply:
It was more of additional check. But for this purpose, the options in the locator search using name are searching for all `accessible names` inside given element. So it could be text, description or label (apparently). In theory, if there is a text inside element, like "Application Navigation" it could differ from aria-label. This we will make sure it is labeled always the same. But I do not insist. it is really additional check in this case.

Reply:
@jeffmaury true.